src/js/api/order.js

```diff
@@ -2456,6 +2456,7 @@ ripe.Ripe.prototype._importOrder = function(ffOrderId, options = {}) {
     const notes = options.notes === undefined ? null : options.notes;
     const images = options.images === undefined ? null : options.images;
     const safe = options.safe === undefined ? null : options.safe ? "1" : "0";
+    const imagesDefault = options.imagesDefault === undefined ? null : options.imagesDefault ? 1 : 0;

```

Comment:
Just like "safe" above, dont we want this to be "1" and "0" (strings instead of integers)

Reply:
RIPE Core is not doing proper casting... "0" casts to `True`

Reply:
Also... We can't elegantly fix it in the core side since all it's doing is `images_default = contents.get("images_default", True)` to later verify with `if not images and images_default:`.
This is the reason why an `images_default` of value `0` is "misinterpreted" as `True`

Reply:
How is it working for "safe" currently?

Reply:
"safe" is not inside `contents` so we can do the proper casting with `safe = self.field("safe", True, cast=bool)`